styles.css

```diff
@@ -188,4 +188,360 @@
     float: left;
     margin-right: 15px;
   }
+
+  /* Works */
+
+  .work {
+    display: grid;
+    grid-template-columns: auto;
+    align-items: center;
+    padding: 114px 24px;
+    gap: 88px;
+  }
+
+  .card-works {
+    display: grid;
```

Comment:
- [ ] Please as demanded by the project requirements, kindly use Flexbox to lay out an individual card.